lib/embedded_ansible.rb

```diff
@@ -153,4 +152,9 @@ def self.generate_password
     SecureRandom.base64(18).tr("+/", "-_")
   end
   private_class_method :generate_password
+
+  def self.connection
+    ActiveRecord::Base.connection
```

Comment:
PGConn has class methods for quoting, but I figured it was better to use AR.

But either way we're also using the connection to create the database. So that's good :smile: